README.md

```diff
@@ -1,2 +1,2 @@
 # Trial-Repository
-# Microverse Trial Michel and Amol
+# Microverse Trial Amol and Richard
```

Comment:
Please Document Project in Professional way as Given in Template by Microverse

Reply:
- [ ] [OPTIONAL] It also ensures the project information like what it does and how its run are easily available to anyone visiting the repo.